recipes-robot/srobo-kit/srobo-kit.bb

```diff
@@ -26,4 +30,12 @@ do_install () {
     install -m 0644 ${WORKDIR}/astoria.toml ${D}/etc/
     install -d ${D}/etc/modprobe.d/
     install -m 0644 ${WORKDIR}/exynos-blacklist.conf ${D}/etc/modprobe.d/
+    install -d ${D}/etc/dnsmasq.d/
+    install -m 0644 ${WORKDIR}/dnsmasq.conf ${D}/etc/dnsmasq.d/srobo.conf
+    install -d ${D}/etc/systemd/network
+    install -m 0644 ${WORKDIR}/br0.netdev ${D}/etc/systemd/network/br0.netdev
+    install -m 0644 ${WORKDIR}/br0.network ${D}/etc/systemd/network/br0.network
+    install -m 0644 ${WORKDIR}/eth0.network ${D}/etc/systemd/network/eth0.network
+    install -d ${D}/etc/sysctl.d
+    echo "net.ipv4.ip_forward=1" > ${D}/etc/sysctl.d/ip_forward.conf
```

Comment:
Nit: Might be worth moving this config to a separate `srobo-network` recipe just to stop the srobo-kit recipe getting too complex.

Reply:
Putting the network config in a separate recipe would seem sensible. given how much of the kit recipe is now networking